src/core/src/op/fake_convert.cpp

```diff
@@ -39,37 +40,28 @@ const std::string& FakeConvert::get_destination_type() const {
 
 void FakeConvert::validate_and_infer_types() {
     OV_OP_SCOPE(v13_FakeConvert_validate_and_infer_types);
-    validate_type();
+    validate_destination_type();
+    auto out_type = ov::element::Type(element::dynamic);
     for (size_t i = 0; i < get_input_size(); i++) {
-        switch (get_input_element_type(i)) {
-        case element::bf16:
-        case element::f16:
-        case element::f32:
-        case element::dynamic:
-            break;
-        default:
-            OPENVINO_THROW("The element type of the input tensor on index ",
-                           i,
-                           " must be a bf16, f16, f32 or dynamic (got ",
-                           get_input_element_type(i),
-                           ").");
-        }
+        OPENVINO_ASSERT(element::Type::merge(out_type, out_type, get_input_element_type(i)),
+                        "Mixed input types are not supported.");
     }
-    if (inputs().size() == 3) {
-        OPENVINO_ASSERT(get_input_partial_shape(1).compatible(get_input_partial_shape(2)),
-                        "FakeConvert scale shape: ",
-                        get_input_partial_shape(1),
-                        " is not compatible with shift shape: ",
-                        get_input_partial_shape(2));
+    switch (out_type) {
+    case element::bf16:
+    case element::f16:
+    case element::f32:
+    case element::dynamic:
+        break;
+    default:
+        OPENVINO_THROW("The element type of the input tensor must be a bf16, f16, f32 or dynamic (got ",
```

Comment:
```suggestion
        OPENVINO_THROW("The element type of the input tensor must be a bf16, f16, f32 but got: " , out_type)
```
Is better to make in exception (message, and value) than (message, value, message) if possible (give same information) takes less space as only join 2 params instead 3

Reply:
Done